x-pack/plugins/transform/public/app/common/request.ts

```diff
@@ -219,6 +219,10 @@ export const getCreateTransformRequestBody = (
     : {}),
   dest: {
     index: transformDetailsState.destinationIndex,
```

Comment:
Did you consider adding the pipeline to the details summary?

![image](https://user-images.githubusercontent.com/7405507/151996409-425dd926-5d4f-4978-a703-e6d65100e6d9.png)